CONTRIBUTING.md

```diff
@@ -37,7 +89,16 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure your commit messages are in the proper format (see below).
 - Push your changes to a topic branch in your fork of the repository.
 - Make sure the tests pass, and add any new tests as appropriate.
-- Submit a pull request to the original repository.
+- Submit a pull request to the original repository and assign it to yourself.
+- Submit a comment with the sole content "@reviewer PTAL" (please take a look) in GitHub
+  and replace "@reviewer" with the correct recipient.
```

Comment:
Fair question and I faced it myself ;-) We have https://github.com/coreos/rkt/blob/master/MAINTAINERS but without any subsystem responsibility.

I have the following proposal:
1. Change the maintainers file in a way that they expose clear subsystem responsibilities (inspired by https://www.kernel.org/doc/linux/MAINTAINERS but smaller)
2. Use the subsystem maintainer as the initial `@reviewer`
3. Define a "dunno" person, i.e. @jonboulle or @alban who can then dispatch to the appropriate person.